otx/algorithms/detection/adapters/mmdet/__init__.py

```diff
@@ -4,7 +4,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 
-from .data import OTXDetDataset
+from .datasets.dataset import OTXDetDataset
```

Comment:
Could you explain why the data folder renamed to datasets folder? I think data folder can cover not only datasets, also the pipelines and other data related utils

Reply:
Modified to match mmdetection's module structure. mmdet uses a module called datasets, including pipeline.